lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -2062,6 +2122,9 @@ bool EarlyFunctionAnalysis<ShouldAnalyzeABI>::runOnModule(Module &M) {
   // Cache pre-population of existing functions in the model
   Properties.importModel();
 
+  // Cache pre-population of imported dynamic symbols
+  Properties.importDynamicSymbols();
```

Comment:
This is part of `importModel`, no reason to split it off.

Reply:
OK, then, since `importDynamicSymbols` iterates over the BBs in `root`, does it make sense to still have its own function `importDynamicSymbols`, but as private member called by `importModel`?

Reply:
You can merge them. You have a bit this tendency to create a new function for each new feature you implement (`importDynamicSymbols` and `initializeFakeFunctions`) but there's not much point to it.

Reply:
Merged!